availability.py

```
@@ -0,0 +1,488 @@
+                    patient=patient,
+                    token_slot__start_datetime__gte=care_now(),
+                )
+                .exclude(status__in=COMPLETED_STATUS_CHOICES)
+                .count()
+                >= settings.MAX_APPOINTMENTS_PER_PATIENT
+            ):
+                error = f"Patient already has maximum number of appointments ({settings.MAX_APPOINTMENTS_PER_PATIENT})"
+                raise ValidationError(error)
+
+            if not patient:
+                raise ValidationError("Patient not found")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)